SU-SWS/stanford_basic
# added core library dependencies

# READY FOR REVIEW

# Summary
- Add normalize and modernize as dependencies 
- This fixes alot of margins and gaps.
- Adds styling for messages so they aren't just plain text.

# Needed By (Date)
- whenever

# Urgency
- low

# Steps to Test

1. spin up new site
2. add the theme and enable
3. ensure the branding bar at the top doesn't have any whitespace around it.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
